### PR TITLE
WLTS: Fix CVEs for which we have a solution in CCD.

### PR DESCRIPTION
**Pull request by mikebrownccd**
### JIRA links (11) ###
https://tools.hmcts.net/jira/browse/WLTS-37
https://tools.hmcts.net/jira/browse/WLTS-38
https://tools.hmcts.net/jira/browse/WLTS-39
https://tools.hmcts.net/jira/browse/WLTS-44
https://tools.hmcts.net/jira/browse/WLTS-45
https://tools.hmcts.net/jira/browse/WLTS-46
https://tools.hmcts.net/jira/browse/WLTS-47
https://tools.hmcts.net/jira/browse/WLTS-48
https://tools.hmcts.net/jira/browse/WLTS-52
https://tools.hmcts.net/jira/browse/WLTS-53
https://tools.hmcts.net/jira/browse/WLTS-57

### Change description ###
Fix 15 CVEs (CVE-2020-5408 , CVE-2022-34305 , CVE-2022-33915 , CVE-2022-31197 , CVE-2022-25857 , CVE-2022-38751 , CVE-2022-38750 , CVE-2022-38749 , CVE-2022-38752 , CVE-2022-42003 , CVE-2022-42004 , CVE-2022-31690 , CVE-2022-31692 , CVE-2022-42252 , CVE-2022-41854).

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dinesh1patel — APPROVED** [review]
LGTM

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -25,52 +26,8 @@
     <gav regex="true">^org\.springframework\.(security|boot):(spring-security-rsa:1\.0\.1[0-1].*|spring-boot-starter-oauth2-(client|resource-server):2\.6\.[0-8])</gav>
     <cve>CVE-2022-22976</cve>
```
Comment (rajaram-nagarajan) [inline config/owasp/suppressions.xml:27]: @JamesCollettCGI  - CVE-2022-22976, CVE-2022-22978 and CVE-2021-22112 are false positives. Can you move these CVEs to false positive section and include the ticket reference?

**Review by rajaram-nagarajan — CHANGES_REQUESTED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (JamesCollettCGI) [reply config/owasp/suppressions.xml:15]: Moved CVE-2022-22976 , CVE-2022-22978 , CVE-2021-22112 to false positives with ticket references.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by dinesh1patel — APPROVED** [review]
LGTM

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**